pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -77,6 +77,8 @@ type TaskRunSpec struct {
 	// +optional
 	// +listType=atomic
 	SidecarOverrides []TaskRunSidecarOverride `json:"sidecarOverrides,omitempty"`
+	// Compute resource configurations applied to the task-level
```

Comment:
suggested docstring: Compute resources to use for this TaskRun

Also, I think this needs to be a pointer to ResourceRequirements, for the same reason as #4913.

Reply:
looked into the details and changes. thanks this ref!